jax-controlnet-sprint/README.md

```diff
@@ -364,3 +366,67 @@ Note, however, that the performance of the TPUs might get bottlenecked as stream
 * [Webdataset](https://webdataset.github.io/webdataset/)
 * [TorchData](https://github.com/pytorch/data)
 * [TensorFlow Datasets](https://www.tensorflow.org/datasets/tfless_tfds)
+
+## How to Make a Submission
+
+To make a full submission, you need to have the following on Hugging Face Hub:
+- Model repository with model weights and model card,
+- Dataset repository with dataset card,
```

Comment:
Is dataset a must? it might not be feasible to create a hf dataset repo if using a large local dataset
should we just ask them to include dataset info in model card for that scenario?

Reply:
I don't think having the dataset should be made mandatory. Instead, we can have that as an expectation. 

> should we just ask them to include dataset info in model card for that scenario?

+1.